pkg/virt-launcher/virtwrap/network/podinterface.go

```diff
@@ -34,11 +36,14 @@ import (
 
 var bridgeFakeIP = "169.254.75.86/32"
 
+// DefaultProtocol is the default port protocol
+const DefaultProtocol string = "TCP"
+
+// DefaultVMCIDR is the default CIRD for vm network
+const DefaultVMCIDR = "10.0.2.0/24"
+
 type BindMechanism interface {
-	discoverPodNetworkInterface() error
```

Comment:
I think you could retain the interface here. We may want to use it later for other types and not reimplement call order / error handling again. Of course, some of these functions will be no-ops in slirp case, but that's ok.

Reply:
@booxter , what other types of networks with pod backend will need these steps?
E.g. if we will implement proxy based approach, the steps are different. 
Do you also suggest to drop the "setup()" method?
If we have an interface with 2 implementations and one of them has empty implementation in all the calls except one, then this is a strong indication that the abstraction is done at the wrong level. 
If there are several types of pod-bridge solution, we can have a BridgePodInterface with these calls - a real interface, not the struct at line 83.

Reply:
Let me implement it with this bind mechanism and make a review about it.

Reply:
Right, there is no disagreement about the current interface being not optimal. But making changing it part of this PR seems to be not appropriate (unless we can't make it work). A separate PR for this specific discussion would be very helpful.